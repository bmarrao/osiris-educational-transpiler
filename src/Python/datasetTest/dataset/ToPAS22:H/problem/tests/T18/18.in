5
4 0
A B C D
6 0
A B C D E F
8 0
A B C D E F G H
10 0
A B C D E F G H I J
12 0
A B C D E F G H I J K L
