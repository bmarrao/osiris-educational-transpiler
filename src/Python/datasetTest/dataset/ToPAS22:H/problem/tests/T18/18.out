Possivel: 24
Possivel: 720
Possivel: 40320
Possivel: 3628800
Possivel: 479001600
